src/main/java/in/projecteka/consentmanager/clients/ErrorMap.java

```diff
@@ -7,21 +7,28 @@
 
 public class ErrorMap {
 
-    private static final Map<Integer, ErrorCode> hipCmErrorMap = new HashMap<> () {
-            {
-                //link
-                put(1000, ErrorCode.NO_PATIENT_FOUND);
-                put(1002, ErrorCode.CARE_CONTEXT_NOT_FOUND);
-                put(1003, ErrorCode.OTP_INVALID);
-                put(1004, ErrorCode.OTP_EXPIRED);
-                put(1006, ErrorCode.INVALID_LINK_REFERENCE);
-                put(1014, ErrorCode.FAILED_TO_GET_LINKED_CARE_CONTEXTS); //FailedToGetLinkedCareContexts
-                put(1015, ErrorCode.DUPLICATE_DISCOVERY_REQUEST); //DuplicateDiscoveryRequest
-            }
+    private static final Map<Integer, ErrorCode> errorMap = new HashMap<>() {
+        {
+            //hip
+            put(3404, ErrorCode.NO_PATIENT_FOUND);
+            put(3402, ErrorCode.CARE_CONTEXT_NOT_FOUND);
+            put(3405, ErrorCode.OTP_INVALID);
+            put(3406, ErrorCode.OTP_EXPIRED);
+            put(3413, ErrorCode.INVALID_LINK_REFERENCE);
+            put(3507, ErrorCode.FAILED_TO_GET_LINKED_CARE_CONTEXTS);
+            put(3409, ErrorCode.DUPLICATE_DISCOVERY_REQUEST);
+            put(3407, ErrorCode.DISCOVERY_REQUEST_NOT_FOUND);
+
+            //gateway
+            put(1000, ErrorCode.UNKNOWN_ERROR_OCCURRED);
```

Comment:
Should these not change? I think you have already changed the Gateway error codes